packages/sql-graphql/test/helper.js

```diff
@@ -126,4 +126,24 @@ module.exports.clear = async function (db, sql) {
     await db.query(sql`DROP TYPE custom_enum`)
   } catch (err) {
   }
+
+  try {
+    await db.query(sql`DROP TABLE test2.books`)
+  } catch (err) {
+  }
+
+  try {
+    await db.query(sql`DROP TABLE test1.authors`)
+  } catch (err) {
+  }
+
+  try {
+    await db.query(sql`DROP SCHEMA test2`)
+  } catch (err) {
+  }
+
+  try {
+    await db.query(sql`DROP SCHEMA test1`)
+  } catch (err) {
+  }
```

Comment:
You should add these to all the other helpers as well, otherwise the tests in the other repos will not pass.

Reply:
Why? I expect every test to clean up its own stuff. Otherwise we should have this in a shared module instead of duplicating the cleaning logic even in modules where these entities (or schema, in this case) are not used.